subjects/isthesquareachild/README.md

```diff
@@ -0,0 +1,48 @@
+## is-the-square-a-child
+
+### Instructions
+
+Write a function `IsTheSquareAChild` that takes a number and returns `true` if the number is a square and `false` otherwise.
+- The function should return `false` if the number is negative.
+- Check only if the first parameter is the square of the second one.
```

Comment:
You ask for this and you program does the opposite...
Your program checks whether the second number is the square of the first and not if the first is the square of the second like you say here ...

Reply:
fixed